website/pages/en/developing/assemblyscript-api.mdx

```diff
@@ -298,6 +298,33 @@ if (transfer == null) {
 
 > Note: If there is no entity created in the given block, `loadInBlock` will return `null` even if there is an entity with the given ID in the store.
 
+#### Looking up derived entities
+
+As of `graph-node` v0.31.0, `@graphprotocol/graph-ts` v0.31.0 and `@graphprotocol/graph-cli` v0.51.0 the `loadRelated` method is available.
```

Comment:
based on this comment it is available https://github.com/graphprotocol/graph-tooling/pull/1190#issuecomment-1533140029 in `graph-node`